### PR TITLE
Fix small issues with passing permissions for UID uploader.

### PR DESCRIPTION
**Pull request by cgc**
review? @rentzso

### CODE REVIEW

File: api/dao/hierarchy.py [hunk 1]
```diff
@@ -383,7 +383,7 @@ def upsert_bottom_up_hierarchy(metadata, user=None, site=None):
         log.error(metadata)
         raise APIStorageException(str(e))
 
-    session_obj = config.db.sessions.find_one({'uid': session_uid}, ['project'])
+    session_obj = config.db.sessions.find_one({'uid': session_uid})
```
Comment (cgc) [inline api/dao/hierarchy.py]: projecting more than just `project` so that the `has_access` check below can access `permissions`

**Comment by cgc** [comment]
@lmperry @rentzso what do you think of this change?

**Comment by gsfr** [comment]
@nagem Could you please take a quick look at this?

@cgc is working with us at Stanford, helping other groups get legacy data into Flywheel.

**Comment by nagem** [comment]
LGTM, will merge now. Thanks for the fix, @cgc.

**Comment by cgc** [comment]
@nagem thanks for mergin! any clue when we can deploy this? in the next few days would be best for my current timeline

**Comment by nagem** [comment]
No problem! Tagging @ryansanford for more info on release schedule.

**Comment by ryansanford** [comment]
Next official Flywheel release is scheduled for next Monday, October 24. I should be cutting a new beta late tomorrow or Thursday. @lmperry is familiar with how to deploy components ahead of release if needed.

**Comment by cgc** [comment]
@ryansanford has this been released?

**Comment by ryansanford** [comment]
@cgc Yes, this change is included in the latest Flywheel release. @lmperry handles the rollout to Stanford systems.

**Pull request merged**